sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -6,6 +6,14 @@ import scalajsbundler.util.{Commands, JS}
 
 object Webpack {
 
+  def copyToWorkingDir(targetDir: File)(file: File): File = {
+    val copy = targetDir / file.name
+    if (copy.getAbsolutePath != file.getAbsolutePath && !copy.exists()) {
```

Comment:
Your're right the first test isn't really necessary and the second is plain wrong (i was assuming an empty target dir). I removed them both.